src/Locator.h

```diff
@@ -19,5 +20,6 @@ namespace openblack
 struct Locator
 {
 	using resources = entt::service_locator<resources::ResourcesInterface>;
+	using rng = entt::service_locator<RNGManager>;
```

Comment:
Create an interface for RNGManager called `RNGManagerInterface`. We don't want to couple the locator to an implementation.